BUG-51188 Update iLoader User's Manual (#616)
ALTIBASE/Documents: Manuals/Altibase_7.1/eng/iLoader User's Manual.md

| \-ssl_cipher *cipher_list*                | Specifies a cipher list for SSL encryption. Please refer to the SSL_CIPHER_LIST property in the *General Reference.* |
| in \| out \| formout \| structout \| help | Sets the direction in which to copy data (it is essential that only one is set)<br/> • in: Copy from a file to a database table. <br/>• out: Copy from a database table to a file. <br/>• formout: Make a table format file (FORM file). <br/>• structout: Performs a function similar to that of formout. Used to create a structure matching that of a given table. (Used when creating a client application.) <br/>• help: Explain how to use help. |
| \-T *table_name*                          | The name of the table to be copied. If you use double quotation marks to enclose this, this is case-sensitive when you execute iLoader in interactive mode. Since the table name is already written in the FORM file, this option is ignored when uploading or downloading data. |

> | \-d *datafile(datafiles)*                 | The full path of the data file used when copying data from the database to the file or vice versa. This is not used with `formout`.<br />`in` Mode:<br />- Up to 32 file names can be specified, and files are uploaded in the listed order.<br />- Each file can upload a maximum of 2,147,483,647 records. If this limit is exceeded, the excess records are ignored, and the upload completes. However, the record count in the log file may not be accurate.<br />`out` Mode:<br />- Each download file can contain up to 2,147,483,647 records. Additional records are downloaded into subsequent files if needed, but the log file may show inaccurate record counts.<br />- With the parallel option, at least as many files as the number specified in the parallel option are created. |

Reviewer: Each download file can contain up to 2,147,483,647 records. Additional records are downloaded into subsequent files if needed
-> subsequent files 부분이 한글 매뉴얼과 좀 다른 것 같습니다. 한계 초과치인 경우도 하나의 파일에 기록됩니다. 다만 로그 파일에 레코드 크기가 부정확하게 기록됩니다.

파일 하나 당 정상적으로 다운로드 할 수 있는 건수는 2,147,483,647건이다. 초과된 데이터는 다운로드할 수 있으나, 로그 파일의 처리 건수 기록에 오류가 발생할 수 있다

Author: ```suggestion
| \-d *datafile(datafiles)*                 | The full path of the data file used when copying data from the database to the file or vice versa. This is not used with `formout`.<br />`in` Mode:<br />- Up to 32 file names can be specified, and files are uploaded in the listed order.<br />- Each file can upload a maximum of 2,147,483,647 records. If this limit is exceeded, the excess records are ignored, and the upload completes. However, the record count in the log file may not be accurate.<br />`out` Mode:<br />- Each file can normally contain up to 2,147,483,647 records for download. If this limit is exceeded, the data can still be downloaded, but the log file may show inaccurate record counts.<br />- With the parallel option, at least as many files as the number specified in the parallel option are created. |
```
해당 내용을 수정했습니다. 문제 없다면 다른 버전의 매뉴얼에도 동일하게 적용하겠습니다~

Reviewer: > 해당 내용을 수정했습니다. 문제 없다면 다른 버전의 매뉴얼에도 동일하게 적용하겠습니다~

빠른 수정 감사드립니다.
